test/java/org/opendatakit/common/cli/ParamTest.java

```diff
@@ -0,0 +1,28 @@
+package org.opendatakit.common.cli;
+
+import org.junit.Test;
+import org.opendatakit.briefcase.reused.BriefcaseException;
+
+import java.time.LocalDate;
+
+import static org.junit.Assert.assertEquals;
+
+public class ParamTest {
+
+  @Test(expected = BriefcaseException.class)
+  public void throw_briefcase_exception_for_invalid_date_format() {
+      Param<LocalDate> localDate = Param.localDate("start", "export_start_date", "Export start date");
+      localDate.map("2018.01.20");
+  }
+
+  @Test
+  public void parse_correctly () {
+      Param<LocalDate> localDate = Param.localDate("start", "export_start_date", "Export start date");
+      // FORMAT yyyy-MM-dd
+      assertEquals(localDate.map("2018-01-20").isEqual(LocalDate.of(2018, 01,20)), true);
+      // FORMAT yyyy/MM/dd
+      assertEquals(localDate.map("2018/01/20").isEqual(LocalDate.of(2018, 01,20)), true);
```

Comment:
Make this an individual test method and make it follow the pattern `assertThat(value, is(expectedValue))`

Name the method according to the behavior that's being asserted. Something like `normalizesInputSlashesToHyphens()`